cvat/apps/tracking/tracker.py

```diff
@@ -83,46 +83,42 @@ def __init__(self, trackerType = "BOOSTING"):
             'MOSSE': cv2.TrackerMOSSE_create,
             'GOTRUN': cv2.TrackerGOTURN_create,
         }
-        if trackerType not in trackerTypes_constructor:
+
+        if trackerType not in self.trackers_constructor:
             raise Exception("Tracker type not known:" + trackerType)
-        self._tracker = trackerTypes_constructor[trackerType]()
 
-    def track_rectangles(self, task, start_shape, stop_frame):
-        """
-        Follow an the rectangle in consecutive frames of a task.
-        :param Task task: The Django Task with the images
-        :param TrackedShape start_shape: Start tracking with this shape; This
-            specifies the frame to start at (start_shape.frame).
-        :param int stop_frame: Stop tracking at this frame (excluded).
-        :return: List of Shapes (Rectangles) with new shapes.
-        """
-        if not isinstance(start_shape, TrackedShape):
-             raise Exception("start_shape must be of type TrackedShape")
+        self._tracker_type = trackerType
 
+    def track_rectangles(self, task, start_shapes, stop_frame):
```

Comment:
what exactly are start_shapes?

Reply:
the first group of shapes that the first frame has and that will be tracked with the multitracker

Reply:
is it their terminology? otherwise I would suggest starting_shapes